src/deep-resource/lib/Resource/Request.js

```diff
@@ -456,6 +457,7 @@ export class Request {
 
       let responseEvent = util._extend({}, requestEvent);
       responseEvent.payload = response;
+      responseEvent.duration = Date.now() - requestEvent.time;
```

Comment:
remove pls `duration` field, it's note allowed by event schema (@see  - https://github.com/MitocGroup/deep-framework/blob/dev/src/deep-log/lib/Driver/RUM/FrameworkEvent.js#L38)